sequencer/interfaces.go

```diff
@@ -43,11 +47,17 @@ type txManager interface {
 
 type workerInterface interface {
```

Comment:
We need to add also "func (w *Worker) AddTx(tx TxTracker)" function that is used by the Pool Loader to load txs from the Pool to the Worker

Reply:
This PR is just for the interfaces needed in the `dbManager` by the `Finalizer`. But after the vacations, as you will be still on vacation I can take a look at adding some of the needed in the `Worker` no probs! :))